drivers/dac/ltc2688/ltc2688.c

```diff
@@ -140,8 +140,16 @@ static int32_t _ltc2688_spi_update_bits(struct ltc2688_dev *dev, uint8_t reg,
  */
 int32_t ltc2688_set_pwr_dac(struct ltc2688_dev *dev, uint16_t setting)
 {
-	return _ltc2688_spi_write(dev, LTC2688_CMD_POWERDOWN_REG,
-				  setting);
+	int32_t ret;
```

Comment:
this should be in a separate commit, it seems unrelated to the dac code change

Reply:
Will add a separate commit for this.

Reply:
Will do it.